# -*- coding: utf-8 -*-
"""
Created on Tue May  7 12:19:52 2019

@author: hp
"""

import os

C:\\Users\\hp\\Desktop\\NishuSharma_python\\07_05_2019
import os

import sys

import calender
Traceback (most recent call last):

  File "<ipython-input-75-02008d9b197e>", line 1, in <module>
    import calender

ModuleNotFoundError: No module named 'calender'




calender
Traceback (most recent call last):

  File "<ipython-input-76-fafb3292a7a4>", line 1, in <module>
    calender

NameError: name 'calender' is not defined




import os

os.name
Out[78]: 'nt'

os.getcwd()
Out[79]: 'C:\\Users\\hp\\Desktop\\NishuSharma_python\\07_05_2019'

os.IP()
Traceback (most recent call last):

  File "<ipython-input-80-fdbc80d439d6>", line 1, in <module>
    os.IP()

AttributeError: module 'os' has no attribute 'IP'




os.path()
Traceback (most recent call last):

  File "<ipython-input-81-74a7c4098921>", line 1, in <module>
    os.path()

TypeError: 'module' object is not callable




os.path
Out[82]: <module 'ntpath' from 'F:\\Users\\hp\\Anaconda3\\lib\\ntpath.py'>

os.listdir()
Out[83]: 
['untitled10.py',
 'untitled11.py',
 'untitled12.py',
 'untitled13.py',
 'untitled14.py']

os.mkdir("abcd")

os.rename("abcd" , "os")

os.rmdir()
Traceback (most recent call last):

  File "<ipython-input-86-4c5594e33bbc>", line 1, in <module>
    os.rmdir()

TypeError: rmdir() missing required argument 'path' (pos 1)




os.rmdir("os")